firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigAutoFetch.java

```diff
@@ -77,6 +77,19 @@ private synchronized void executeAllListenerCallbacks() {
     }
   }
 
+  private String parseMessage(String message) {
```

Comment:
For our[ parsing](https://github.com/firebase/firebase-android-sdk/blob/6150f7b28cbeff3ba81d98d29bc6419b1a1cd43c/firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigAutoFetch.java#L117), since it's a stream there will be multiple messages coming through our while loop so we can't just end it when we get one message. So what we do is add every incoming message to our string fullMessage string. When we see a message that contains '}' then we know the backend just finished sending a full message. We then call parseMessage which just strips the beginning and end of fullMessage up until it reaches '{' and '}'. Everything between '{' and. '}' should be properly formatted so we don't need to do anything there.